src/platform/sdl/main.c

```diff
@@ -321,3 +337,61 @@ static void mSDLDeinit(struct mSDLRenderer* renderer) {
 
 	SDL_Quit();
 }
+
+void setLogger(struct mCore* core) {
+	int fakeBool = 0;
+	bool logToStdout = true;
+	bool logToFile = false;
+
+	if (mCoreConfigGetIntValue(&core->config, "logToStdout", &fakeBool)) {
+		logToStdout = fakeBool;
+	}
+	if (mCoreConfigGetIntValue(&core->config, "logToFile", &fakeBool)) {
+		logToFile = fakeBool;
+	}
+	const char* logFile = mCoreConfigGetValue(&core->config, "logFile");
+
+	// Assign basic static variables
+	_logToStdout = logToStdout;
+	_logFile = NULL;
+	
+	if(logToFile && logFile) {
+		_logFile = VFileOpen(logFile, O_WRONLY | O_CREAT | O_APPEND);
+	}
+
+	// Create the filter
+	mLogFilterInit(&_filter);
+	mLogFilterSet(&_filter, "gba.bios", mLOG_FATAL);
+	mLogFilterSet(&_filter, "core.status", mLOG_ALL & ~mLOG_DEBUG);
```

Comment:
The point of this was so that status messages bubble up into the OSD on the Qt version. However, there is no OSD in the SDL version, so this might create spam for users by default. I'm unsure if that's a _problem_ though.